common/helpers/date.js

```diff
@@ -7,23 +9,21 @@ const {
   endOfWeek,
   isValid: isValidDate,
   isDate,
+  isSameWeek,
 } = require('date-fns')
 
 const { DATE_FORMATS } = require('../../config/index')
+const DEFAULT_DATE_OPTIONS = { weekStartsOn: DATE_FORMATS.WEEK_STARTS_ON }
```

Comment:
All the functions were sharing the same date options, it made sense to pull them out here